pyproject.toml

```diff
@@ -0,0 +1,74 @@
+[tool.poetry]
+name = "pelican-jinja-filters"
+version = "1.0.4"
+description = "Pelican plugin for applying useful Jinja filters in theme templates"
+authors = ["William Minchin <w_minchin@hotmail.com>", "Justin Mayer <entroP@gmail.com>"]
+license = "MIT"
+readme = "README.rst"
+keywords = ["jinja2", "templates", "pelican", "plugin"]
+repository = "https://github.com/pelican-plugins/jinja-filters"
+documentation = "https://docs.getpelican.com"
+packages = [
+    { include = "pelican" },
+]
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Framework :: Pelican",
+    "Framework :: Pelican :: Plugins",
+    "Intended Audience :: End Users/Desktop",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Topic :: Internet :: WWW/HTTP",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+
+[tool.poetry.urls]
+"Funding" = "https://donate.getpelican.com/"
+"Tracker" = "https://github.com/pelican-plugins/jinja-filters/issues"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+pelican = "^4.2"
+markdown = {version = "^3.2.2",optional = true}
```

Comment:
I don't think we need `markdown`...

Reply:
I usually include this for all Pelican-related tooling since it's a frequently-used package, and for plugins that include tests, `markdown` is indeed sometimes required. That's why it's added here as an *optional* dependency. That said, this can probably be removed if you prefer.